js/main.js

```diff
@@ -0,0 +1,207 @@
+  for (var i = Math.floor(Math.random() * exercises.quads.length); numExercises< 2; i++){
+    var $li = document.createElement('li');
+    $li.setAttribute('id', exercises.quads[i].uuid);
+    var $p = document.createElement('p');
+    $p.textContent = exercises.quads[i].name;
+    $li.appendChild($p)
+    $quadsList.appendChild($li);
+    numExercises++;
+      }
+}
+
+$legsButton.addEventListener('click', renderQuads);
+
+//-----------Render Glutes---------------
+
+function renderGlutes(){
+  var numExercises = 0;
+  for (var i = Math.floor(Math.random() * exercises.glutes.length); numExercises< 2; i++){
+    var $li = document.createElement('li');
+    $li.setAttribute('id', exercises.glutes[i].uuid);
+    var $p = document.createElement('p');
+    $p.textContent = exercises.glutes[i].name;
+    $li.appendChild($p)
+    $glutesList.appendChild($li);
+    numExercises++;
+      }
+}
+
+$legsButton.addEventListener('click', renderGlutes);
+
+//-----------Render Hams------------------
+
+function renderHams(){
+  var numExercises = 0;
+  for (var i = Math.floor(Math.random() * exercises.hams.length); numExercises< 2; i++){
+    var $li = document.createElement('li');
+    $li.setAttribute('id', exercises.hams[i].uuid);
+    var $p = document.createElement('p');
+    $p.textContent = exercises.hams[i].name;
+    $li.appendChild($p)
+    $hamsList.appendChild($li);
+    numExercises++;
+      }
+}
+
+$legsButton.addEventListener('click', renderHams);
+
+//-----------Render Traps------------------
+
+function renderTraps(){
+  var numExercises = 0;
+  for (var i = Math.floor(Math.random() * exercises.traps.length); numExercises< 2; i++){
+    var $li = document.createElement('li');
+    $li.setAttribute('id', exercises.traps[i].uuid);
+    var $p = document.createElement('p');
+    $p.textContent = exercises.traps[i].name;
+    $li.appendChild($p)
+    $trapsList.appendChild($li);
+    numExercises++;
+      }
+}
+
+$pullButton.addEventListener('click', renderTraps);
+
+//-----------Render Lats-------------------
+
+function renderLats(){
+  var numExercises = 0;
+  for (var i = Math.floor(Math.random() * exercises.lats.length); numExercises< 2; i++){
+    var $li = document.createElement('li');
+    $li.setAttribute('id', exercises.lats[i].uuid);
+    var $p = document.createElement('p');
+    $p.textContent = exercises.lats[i].name;
+    $li.appendChild($p)
+    $latsList.appendChild($li);
+    numExercises++;
+      }
+}
+
+$pullButton.addEventListener('click', renderLats);
+
+//-----------Render Biceps------------------
+
+function renderBiceps(){
+  var numExercises = 0;
+  for (var i = Math.floor(Math.random() * exercises.biceps.length); numExercises< 2; i++){
+    var $li = document.createElement('li');
+    $li.setAttribute('id', exercises.biceps[i].uuid);
+    var $p = document.createElement('p');
+    $p.textContent = exercises.biceps[i].name;
+    $li.appendChild($p)
+    $bicepsList.appendChild($li);
+    numExercises++;
+      }
+}
```

Comment:
Rather than making one function for each type, you should make one function that receives the name of the property from `exercises` to use. You could store it in a `data-muscle-group` attribute on whatever element is being clicked to start this behavior, then retrieve that value, and call the function with it as an argument.